src/elias_fano/tests.rs

```diff
@@ -223,20 +221,17 @@ fn test_iter() {
     assert_eq!(iter.next(), None);
     assert_eq!(iter.next_back(), None);
     assert_eq!(iter.next(), None);
-    drop(iter); // end borrow for next test
 
     // owning iter and into_iter test
-    let mut i = 0;
-    for elem in ef {
-        assert_eq!(elem, i);
-        i += 1;
+    for (i, elem) in ef.into_iter().enumerate() {
+        assert_eq!(elem, i as u64);
     }
 }
 
 #[test]
 fn test_custom_iter_behavior() {
-    let ef = EliasFanoVec::from_slice(&vec![0, 1, 2, 3, 4, 5, 6, 7, 8]);
-    assert_eq!(ef.iter().skip(2).next(), Some(2));
```

Comment:
Same here: skip().next() is called intentionally to test the skip function over nth(). Same a few lines below.